pyproject.toml

```diff
@@ -12,8 +12,8 @@ keywords = ["validation", "dataframe"]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-pydantic = ">=2.4.1"
-polars = ">=0.20.1"
+pydantic = ">=2.4.1 <2.8"
```

Comment:
Hi, is there a reason for this constraint to be added on pydantic?

Reply:
> Hi, is there a reason for this constraint to be added on `pydantic`?
> 
> Also do we have any estimation about the time to merge this PR as polars v1.0.0 has been released? Thanks for the work.

A param changed in derive model after 2.8, and I needed a quick fix to unblock

Reply:
Fixed.